README.md

```diff
@@ -113,11 +108,6 @@ Licensed under either of
  * MIT license ([LICENSE-MIT](LICENSE-MIT) or
    http://opensource.org/licenses/MIT) at your option.
 
-### Acknowledgements
```

Comment:
I would not completely remove this, maybe just change the wording. I think pyOCD was pretty essential for the initial flash code, so I would keep an acknowledgement section.